amm/src/main/scala/ammonite/interp/Preprocessor.scala

```diff
@@ -289,16 +304,16 @@ object Preprocessor{
                printCode: String,
                imports: Imports) = {
 
-    val topWrapper = s"""
+    val topWrapper = normalizeNewlines(s"""
 package ${pkgName.map(_.backticked).mkString(".")}
 ${importBlock(imports)}
 
-object ${indexedWrapperName.backticked}{\n"""
+object ${indexedWrapperName.backticked}{\n""")
 
-    val bottomWrapper = s"""\ndef $$main() = { $printCode }
+    val bottomWrapper = normalizeNewlines(s"""\ndef $$main() = { $printCode }
   override def toString = "${indexedWrapperName.raw}"
 }
-"""
+""")
```

Comment:
On windows also it will remain '\n' only, though it does not affect the line numbers or anything else. I thought it would be better to avoid wrong newline chars on windows as `top(bottom)Wrapper` gets appended as it is to the code while code already has `\r\n` on windows.